### ISSUE TITLE
Flux Controlnet optional_mask support?

### ISSUE THREAD

**Issue by yinian-lw** [Kosinkadink/ComfyUI-Advanced-ControlNet#171]

**Comment by Satoshi-Yoda** [comment]
Yes, I am having the same issue. Masked control net is a great feature!
At the moment it fails with an error
```The size of tensor a (12555) must match the size of tensor b (12420) at non-singleton dimension 1```
if I try to specify `mask_optional`. Without `mask_optional` works still great! Would be perfect to have such option available with flux also.
